WcaOnRails/app/models/user.rb

```diff
@@ -65,7 +65,7 @@ def wca_id_is_unique_or_for_dummy_account
       user = User.find_by_wca_id(wca_id)
       # If there is a non dummy user with this WCA ID, fail validation.
       if user && !user.dummy_account?
-        errors.add(:wca_id, I18n.t('users.errors.unique'))
+        errors.add(:wca_id, I18n.t('users.errors.unique', email: user.email))
```

Comment:
Oops just so @jfly's comment. No idea, I totally didn't think about that :s

I'm not sure changing the key would help, because in order to make the tests pass one would have to change the keys in other locales too, without having any idea of what the new translation should be.

I created #847 to keep track of this, I'll think more about this later, hopefully someone will have ideas too :)